src/Navigation/Navigation.Xamarin.Forms/Service/NavigationService.show.cs

```diff
@@ -196,46 +167,23 @@ internal async Task<TViewModelInterface> InitializeViewModelInternal<TViewModelI
         {
             if (!TryGetRegistration(
                 typeof(TViewModel),
-                out var registrationInfo))
+                out var registration))
             {
                 // todo: set exception message
                 throw new InvalidOperationException();
             }
 
 
-            if (registrationInfo.ViewType.IsAssignableFrom(typeof(TPage)))
+            if (registration.ViewType.IsAssignableFrom(typeof(TPage)))
             {
                 throw new InvalidOperationException(
-                    $"Non-assignable view type specifications for ViewModel '{typeof(TViewModel).Name}' - registered: '{registrationInfo.ViewType.Name}', generic parameter: {typeof(TPage).Name}");
+                    $"Non-assignable view type specifications for ViewModel '{typeof(TViewModel).Name}' - registered: '{registration.ViewType.Name}', generic parameter: {typeof(TPage).Name}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 192 characters long line (which is greater than 120 authorized).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)